Frontend/src/pages/registration_administration/components/StatusDropdown.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export default function StatusDropdown({ status, setStatus }) {
+  const options = ['waiting', 'accepted', 'deleted']
```

Comment:
It's nice to pull constants like this out of the component, otherwise a fresh array & reference is created each render.